### PR TITLE
Release/2.8.0

### PR DESCRIPTION
**Pull request by eugene-manuilov**
### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [x] All new and existing tests passed.

### Known Issues

- [x] A question to empty the default content isn't asked when a new site is being created
- [x] When docker-compose throws an error, the spinner keeps working and doesn't display an error

### CODE REVIEW

**Comment by nicholasio** [comment]
I've been using 2.8.0 for the last couple of months and it's working great.

From a QA/Testing perspective, you have my 👍

File: package.json
Comment (nicholasio) [inline package.json:89]: It would be great to update this to 2.x at some point.

**Review by nicholasio — APPROVED** [review]
Did a quick-pass and code-wise things look good.

File: package.json
Comment (jeffpaul) [reply package.json:89]: @eugene-manuilov @nicholasio any reason not to do that now?  No reason this would affect the 2.8 release as far as I could image.

Comment (nicholasio) [reply package.json:89]: the rules changed quite a bit, there might be things that can't be fixed automatically. I'll defer to @eugene-manuilov

Comment (jeffpaul) [reply package.json:89]: @eugene-manuilov seems like holding off on bumping `@10up/eslint-config` to 2.x makes most sense so as to not further delay the 2.8.0 release, yeah?

Comment (eugene-manuilov) [reply package.json:89]: Yeah, it's not critical for the 2.8.0 release and can be done later.

**Comment by jeffpaul** [comment]
@christianc1 @tylercherpak @dustinrue any chance you all can review this week in hopes of releasing 2.8 this/next week?

**Comment by jeffpaul** [comment]
@christianc1 @tylercherpak either of you good for a 👍🏼 on releasing 2.8.0 now?

**Pull request merged**
